src/server.c

```diff
@@ -3099,6 +3099,7 @@ struct redisCommand *lookupSubcommand(struct redisCommand *container, sds sub_na
  * a user requested to execute (in processCommand).
  */
 struct redisCommand *lookupCommandLogic(dict *commands, robj **argv, int argc, int strict) {
+    serverAssert(argc != 0);
```

Comment:
what is that about?

Reply:
warning in GCC-13: https://github.com/sundb/redis/actions/runs/4718286875/jobs/8367719506